.github/copilot-instructions.md

```diff
@@ -0,0 +1,114 @@
+# .NET IoT — Repository Custom Instructions for GitHub Copilot
+
+> Purpose: steer Copilot to propose changes that match this repo’s structure, coding conventions, and contribution norms for **System.Device.Gpio** and **Iot.Device.Bindings**.
+
+---
+
+## Scope & context (what this repo is)
+- This repo contains the low-level **System.Device.Gpio** APIs and a large collection of device drivers under **Iot.Device.Bindings**.
+- Code is used across different OSes and boards. Prefer **hardware-agnostic** code, with platform-specific behavior isolated behind drivers/abstractions.
+- Treat public APIs as **experimental but stable-in-spirit**: prefer additive changes, avoid breaking patterns unless maintainers direct otherwise.
+
+---
+
+## Build, test, and layout (how to work here)
+- **Solution layout:** library code lives under `src/` (e.g., `src/System.Device.Gpio`, `src/devices/<DeviceName>`); runnable examples live under `samples/` and often under each device’s folder.
+- **Build:** default to `dotnet build` at the root or use the repo scripts (`build.sh`/`Build.cmd`) when appropriate. Respect TFMs and settings defined by `Directory.Build.*` and `.editorconfig`.
```

Comment:
I would suggest being more emphatic into using the build scripts from the repo here. That will guarantee that the agents will use the right tools.